styles.css

```diff
@@ -1,504 +1,1051 @@
-body {
-  background: #e5e5e5;
-  margin: 0;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
+@media only screen and (max-width: 767px) {
+  body {
+    background: #e5e5e5;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
 
-.header {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  padding-top: 25px;
-  width: 375px;
-}
+  .header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 25px;
+    width: 375px;
+  }
```

Comment:
- [ ] Although you have done a commendable job with the design of your project :+1:, it is necessary to closely adhere to the Figma template. Therefore, I suggest that you review the arrangement of the header in relation to the main section to ensure that it is accurately placed. Currently, it appears that your logo is positioned more towards the right compared to the logo in the template, and I kindly request that you rectify this problem :+1:.

**Your design**

![Screenshot from 2023-03-31 18-12-29](https://user-images.githubusercontent.com/105117832/229248769-d2cf62a5-f3c4-4675-8c9b-44cb4775dea2.png)

**Figma**

![Screenshot from 2023-03-31 18-12-40](https://user-images.githubusercontent.com/105117832/229248749-71b93c09-02b5-4f53-a011-4b87dca18842.png)